azureml_scripts/manage_compute_instance.py

```diff
@@ -0,0 +1,142 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+import argparse
+from pathlib import Path
+
+try:
+    from azure.ai.ml import MLClient
+    from azure.ai.ml.entities import AmlCompute
+except ImportError:
+    raise ImportError("azure-ai-ml is not installed. Please install azure-ai-ml packages to use this script.")
+
+try:
+    from azure.identity import AzureCliCredential, DefaultAzureCredential, InteractiveBrowserCredential
+except ImportError:
+    raise ImportError("azure-identity is not installed. Please install azure-identity packages to use this script.")
+
+
+def get_args():
```

Comment:
>get_args [](http://example.com/codeflow?start=4&length=8)

need we put this script into olive wheel?

Reply:
Good point. Added to Olive wheel.